README.md

```diff
@@ -102,69 +102,104 @@ mocha -R mocha-multi --reporter-options spec=-,json=.results.json --timeout 1000
+    4) takes a string argument and logs it in all lowercase using console.log()
 
   sayHiToGrandma(string)
     5) returns "I can't hear you!" if `string` is lowercase
     6) returns "YES INDEED!" if `string` is uppercase
     7) returns "I love you, too." if `string` is "I love you, Grandma."`
 
 
-  0 passing (99ms)
+  0 passing (141ms)
   7 failing
 
   1) shout(string)
        receives one argument and returns it in all caps:
      ReferenceError: shout is not defined
       at Context.<anonymous> (test/index-test.js:4:5)
-      at processImmediate (internal/timers.js:456:21)
+      at processImmediate (internal/timers.js:461:21)
 
   2) whisper(string)
        receives one argument and returns it in all lowercase:
      ReferenceError: whisper is not defined
       at Context.<anonymous> (test/index-test.js:10:5)
-      at processImmediate (internal/timers.js:456:21)
+      at processImmediate (internal/timers.js:461:21)
 
   3) logShout(string)
-       calls console.log() its one argument in all caps:
+       takes a string argument and logs it in all caps using console.log():
      ReferenceError: logShout is not defined
       at Context.<anonymous> (test/index-test.js:18:5)
-      at processImmediate (internal/timers.js:456:21)
+      at processImmediate (internal/timers.js:461:21)
 
   4) logWhisper(string)
-       calls console.log() its one argument in all lowercase:
+       takes a string argument and logs it in all lowercase using console.log():
      ReferenceError: logWhisper is not defined
       at Context.<anonymous> (test/index-test.js:30:5)
-      at processImmediate (internal/timers.js:456:21)
+      at processImmediate (internal/timers.js:461:21)
 
   5) sayHiToGrandma(string)
        returns "I can't hear you!" if `string` is lowercase:
      ReferenceError: sayHiToGrandma is not defined
       at Context.<anonymous> (test/index-test.js:40:5)
-      at processImmediate (internal/timers.js:456:21)
+      at processImmediate (internal/timers.js:461:21)
 
   6) sayHiToGrandma(string)
        returns "YES INDEED!" if `string` is uppercase:
      ReferenceError: sayHiToGrandma is not defined
       at Context.<anonymous> (test/index-test.js:44:5)
-      at processImmediate (internal/timers.js:456:21)
+      at processImmediate (internal/timers.js:461:21)
 
   7) sayHiToGrandma(string)
        returns "I love you, too." if `string` is "I love you, Grandma."`:
      ReferenceError: sayHiToGrandma is not defined
       at Context.<anonymous> (test/index-test.js:48:5)
-      at processImmediate (internal/timers.js:456:21)
+      at processImmediate (internal/timers.j
+```
 
+Hm, seven failed tests - that seems like a lot to navigate. Let's go ahead and
+get Mocha set up to only show the first failing test. First, create the
+`mocha.opts` file:
 
+```sh
+touch test/mocha.opts
+```
 
-npm ERR! Test failed.  See above for more details.
+Then open the new file in your text editor and add the following:
+
+```text
+--bail
 ```
```

Comment:
Adding `--bail` at the end of the script in the `package.json` file also does the trick and might be easier for students to manage. That like would look like this:

```
"test": "mocha --timeout 5000 --reporter 'json' > .results.json & mocha --bail"
```

(Aside: I forgot that we changed from using mocha-multi for some of the prework labs, so this script is a bit different than what they'd see in Phase 1... it technically runs mocha twice, the first time to generate the json report and the second time to show the test output in the terminal)

I tried this out on a Phase 1 lab using `learn test` and everything seemed to work ok, so it looks like this is a nice workflow to give students!